guides/common/modules/proc_enabling-client-connections-to-satellite.adoc

```diff
@@ -9,31 +9,33 @@ For more information on the ports used, see {InstallingServerDocURL}satellite-po
 include::snip_firewalld.adoc[]
 
 .Procedure
-. To open the ports for client to {Project} communication, enter the following command on the base operating system that you want to install {Project} on:
+Enter the following commands on the base operating system that you want to install {Project} on.
+
```

Comment:
This sentence here doesn't adhere to our formatting standards and it looks bad. Also, I don't think it's necessary, because the same is said right before the procedure. Can we remove it?